diff -urw gr-osmosdr-0.1.4.9.1/lib/xtrx/xtrx_obj.cc src/gr-osmosdr-0.1.4.9.1/lib/xtrx/xtrx_obj.cc
--- gr-osmosdr-0.1.4.9.1/lib/xtrx/xtrx_obj.cc	2019-08-10 23:03:00.790000000 +0000
+++ src/gr-osmosdr-0.1.4.9.1/lib/xtrx/xtrx_obj.cc	2019-08-10 23:03:31.280000000 +0000
@@ -68,7 +68,7 @@
   unsigned xtrxflag = (loglevel & XTRX_O_LOGLVL_MASK) | ((lmsreset) ? XTRX_O_RESET : 0);
   std::cerr << "xtrx_obj::xtrx_obj = " << xtrxflag << std::endl;
 
-  int res = xtrx_open_list(path.c_str(), NULL, &_obj);
+  int res = xtrx_open_string(path.c_str(), &_obj);
   if (res < 0) {
     std::stringstream message;
     message << "Couldn't open "  ": Error: " << -res;
