diff -Nurw compute-runtime-20.08.15750/unit_tests/test_files/vme_kernels.cl src/compute-runtime-20.08.15750/unit_tests/test_files/vme_kernels.cl
--- compute-runtime-20.08.15750/unit_tests/test_files/vme_kernels.cl	2020-02-29 18:46:48.088040001 +0000
+++ src/compute-runtime-20.08.15750/unit_tests/test_files/vme_kernels.cl	2020-02-29 18:47:01.318040617 +0000
@@ -41,6 +41,15 @@
 
 \*************************************************************************************************/
 
+#pragma OPENCL EXTENSION cl_intel_subgroups : enable
+#pragma OPENCL EXTENSION cl_intel_device_side_avc_motion_estimation : enable
+#ifndef CLK_AVC_ME_SEARCH_WINDOW_16x12_RADIUS_INTEL
+#define CLK_AVC_ME_SEARCH_WINDOW_16x12_RADIUS_INTEL        0x9
+#endif
+#ifndef NULL
+#define NULL 0L
+#endif
+
 __kernel __attribute__((reqd_work_group_size(16, 1, 1)))
 void device_side_block_motion_estimate_intel(__read_only image2d_t srcImg,
                                              __read_only image2d_t refImg,
