diff -Nurw compute-runtime-20.08.15750/core/os_interface/linux/os_library_linux.cpp src/compute-runtime-20.08.15750/core/os_interface/linux/os_library_linux.cpp
--- compute-runtime-20.08.15750/core/os_interface/linux/os_library_linux.cpp	2020-03-04 16:06:31.458542632 +0000
+++ src/compute-runtime-20.08.15750/core/os_interface/linux/os_library_linux.cpp	2020-03-04 16:07:02.031877363 +0000
@@ -30,7 +30,7 @@
         this->handle = dlopen(0, RTLD_LAZY);
     } else {
 #ifdef SANITIZER_BUILD
-        constexpr auto dlopenFlag = RTLD_LAZY;
+        constexpr auto dlopenFlag = RTLD_LAZY | RTLD_GLOBAL;
 #else
         constexpr auto dlopenFlag = RTLD_LAZY | RTLD_DEEPBIND;
 #endif
