diff -Nurw compute-runtime-20.08.15750/unit_tests/built_ins/built_in_kernels_tests.cpp src/compute-runtime-20.08.15750/unit_tests/built_ins/built_in_kernels_tests.cpp
--- compute-runtime-20.08.15750/unit_tests/built_ins/built_in_kernels_tests.cpp	2020-02-29 17:38:55.007853054 +0000
+++ src/compute-runtime-20.08.15750/unit_tests/built_ins/built_in_kernels_tests.cpp	2020-02-29 17:39:09.864520402 +0000
@@ -6,6 +6,7 @@
  */
 
 #include "runtime/builtin_kernels_simulation/opencl_c.h"
+#undef select
 
 #include "gtest/gtest.h"
 
diff -Nurw compute-runtime-20.08.15750/unit_tests/command_queue/get_size_required_buffer_tests.cpp src/compute-runtime-20.08.15750/unit_tests/command_queue/get_size_required_buffer_tests.cpp
--- compute-runtime-20.08.15750/unit_tests/command_queue/get_size_required_buffer_tests.cpp	2020-02-29 18:21:26.794636727 +0000
+++ src/compute-runtime-20.08.15750/unit_tests/command_queue/get_size_required_buffer_tests.cpp	2020-02-29 18:22:17.897972415 +0000
@@ -9,6 +9,7 @@
 #include "runtime/command_queue/enqueue_fill_buffer.h"
 #include "runtime/command_queue/enqueue_kernel.h"
 #include "runtime/command_queue/enqueue_read_buffer.h"
+#undef select
 #include "runtime/command_queue/enqueue_write_buffer.h"
 #include "runtime/command_queue/gpgpu_walker.h"
 #include "runtime/event/event.h"
