diff -urw src/openscad-2019.05/features/cgal.prf openscad-2019.05/features/cgal.prf
--- src/openscad-2019.05/features/cgal.prf	2018-12-05 19:50:12.000000000 +0000
+++ openscad-2019.05/features/cgal.prf	2020-01-12 20:21:07.680000000 +0000
@@ -9,7 +9,7 @@
   message("CGAL location: $$CGAL_DIR")
 }
 
-LIBS += -lCGAL -lmpfr -lgmp
+LIBS += -lmpfr -lgmp
 
 *g++* {
   QMAKE_CXXFLAGS += -frounding-math
diff -urw src/openscad-2019.05/src/parser.y openscad-2019.05/src/parser.y
--- src/openscad-2019.05/src/parser.y	2019-04-07 15:45:27.000000000 +0000
+++ openscad-2019.05/src/parser.y	2020-01-12 20:20:16.010000000 +0000
@@ -48,6 +48,7 @@
 #include <sstream>
 #include <boost/filesystem.hpp>
 #include "boost-utils.h"
+#include <stack>
 
 namespace fs = boost::filesystem;
 
diff -urw src/openscad-2019.05/src/printutils.h openscad-2019.05/src/printutils.h
--- src/openscad-2019.05/src/printutils.h	2019-03-09 21:41:45.000000000 +0000
+++ openscad-2019.05/src/printutils.h	2020-01-12 20:20:43.070000000 +0000
@@ -45,15 +45,15 @@
 void printDeprecation(const std::string &str);
 void resetSuppressedMessages();
 
-#define PRINT_DEPRECATION(_fmt, _arg) do { printDeprecation(str(boost::format(_fmt) % _arg)); } while (0)
+#define PRINT_DEPRECATION(_fmt, _arg) do { printDeprecation(boost::str(boost::format(_fmt) % _arg)); } while (0)
 
 /* PRINT statements come out in same window as ECHO.
  usage: PRINTB("Var1: %s Var2: %i", var1 % var2 ); */
 void PRINT(const std::string &msg);
-#define PRINTB(_fmt, _arg) do { PRINT(str(boost::format(_fmt) % _arg)); } while (0)
+#define PRINTB(_fmt, _arg) do { PRINT(boost::str(boost::format(_fmt) % _arg)); } while (0)
 
 void PRINT_NOCACHE(const std::string &msg);
-#define PRINTB_NOCACHE(_fmt, _arg) do { PRINT_NOCACHE(str(boost::format(_fmt) % _arg)); } while (0)
+#define PRINTB_NOCACHE(_fmt, _arg) do { PRINT_NOCACHE(boost::str(boost::format(_fmt) % _arg)); } while (0)
 
 void PRINT_CONTEXT(const class Context *ctx, const class Module *mod, const class ModuleInstantiation *inst);
 
