diff -Nur src/serialDV-1.1.1/udpdatacontroller.cpp serialDV-1.1.1/udpdatacontroller.cpp
--- src/serialDV-1.1.1/udpdatacontroller.cpp	2019-07-29 09:15:30.000000000 +0000
+++ serialDV-1.1.1/udpdatacontroller.cpp	2019-08-16 16:02:41.200000000 +0000
@@ -23,11 +23,13 @@
 #include <WS2tcpip.h>
 #pragma comment(lib, "Ws2_32.lib")
 #else
+#include <sys/select.h>
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <unistd.h>
 #include <string.h>
+#include <time.h>
 #endif
 
 #include "udpdatacontroller.h"
@@ -232,4 +234,4 @@
     }
 }
 
-} // namespace SerialDV
\ No newline at end of file
+} // namespace SerialDV
