diff -urw src/SoapySDR-soapy-sdr-0.7.1/cmake/Modules/SoapySDRConfig.cmake new/SoapySDR-soapy-sdr-0.7.1/cmake/Modules/SoapySDRConfig.cmake
--- SoapySDR-soapy-sdr-0.7.1/cmake/Modules/SoapySDRConfig.cmake	2019-01-26 18:14:47.000000000 +0000
+++ SoapySDR-soapy-sdr-0.7.1/cmake/Modules/SoapySDRConfig.cmake	2020-01-09 22:28:35.540000000 +0000
@@ -74,10 +74,6 @@
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
     endif()
 
-    #force a compile-time error when symbols are missing
-    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
-    set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -Wl,--no-undefined")
-
     #common warnings to help encourage good coding practices
     add_compile_options(-Wall)
     add_compile_options(-Wextra)
