diff -urw src/libosmocore-1.3.0/git-version-gen libosmocore-1.3.0/git-version-gen
--- src/libosmocore-1.3.0/git-version-gen	2020-01-02 17:42:30.000000000 +0000
+++ libosmocore-1.3.0/git-version-gen	2020-01-11 00:46:38.470000000 +0000
@@ -130,7 +130,7 @@
 # Don't declare a version "dirty" merely because a time stamp has changed.
 git status > /dev/null 2>&1
 
-dirty=`sh -c 'git diff-index --name-only HEAD' 2>/dev/null` || dirty=
+#dirty=`sh -c 'git diff-index --name-only HEAD' 2>/dev/null` || dirty=
 case "$dirty" in
     '') ;;
     *) # Append the suffix only if there isn't one already.
